src/youtube-player/youtube-player.ts

```diff
@@ -157,6 +158,17 @@ export class YouTubePlayer implements AfterViewInit, OnDestroy, OnInit {
   }
   private _suggestedQuality = new BehaviorSubject<YT.SuggestedVideoQuality | undefined>(undefined);
 
+  /**
+   * Extra parameters used to configure the player. See:
+   * https://developers.google.com/youtube/player_parameters.html?playerVersion=HTML5#Parameters
+   */
+  @Input()
+  get playerVars(): YT.PlayerVars | undefined { return this._playerVars.value; }
+  set playerVars(playerVars: YT.PlayerVars | undefined) {
```

Comment:
@jelbourn In https://github.com/angular/components/pull/17672#discussion_r362396678 I was advocating against passing in the `playerVars` object to the input. I changed my mind, because the object has a lot of properties that we'd have to proxy which will increase the API surface a lot. I'm open to changing it if you think otherwise.

Reply:
That's fair. It's a little less Angular-y, but adding all those APIs will be bad for payload size.

Reply:
I think the internals of the player are due for a rework anyway since this reliance on an observable + getter + setter for every single input isn't scalable.